### DISCUSSION TITLE
[@types/uglify-js] Add `webkit` to CompressOptions

### DISCUSSION THREAD

**Discussion by NoelAbrahams** [DefinitelyTyped/DefinitelyTyped#59478] [Issues with a @types package]
The following needs to be added to interface [`CompressOptions`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/87fed5c5355e4518db03253847a1ce0cd40c05db/types/uglify-js/index.d.ts#L25).

```ts

/** 
 * Support non-standard Safari/Webkit.
 * By default UglifyJS will not try to be Safari-proof.
 */
webkit?: boolean;

```

See https://github.com/mishoo/UglifyJS/issues/5032#issuecomment-867094768.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
fixing: #59482
